enhancements/installer/kubernete-slb-owned-api-loadbalancers.md

```diff
@@ -0,0 +1,508 @@
+---
+title: kubernetes-slb-owned-api-loadbalancers
+authors:
+  - "@abhinavdahiya"
+reviewers:
+  - "@sttts"
+  - "@smarterclayton"
+approvers:
+  - "@sttts"
+  - "@smarterclayton"
+creation-date: 2020-05-06
+last-updated: 2020-08-26
+status: implementable
+see-also:
+  -   
+replaces:
+  - 
+superseded-by:
+  - 
+---
+
+# Kubernetes Service Owned API Loadbalancers
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+1. Service type Loadbalancer's ExternalTrafficPolicy should be Cluster or Local ?
+2. How to reserve private IPs on AWS ?
+
+## Summary
+
+Currently the installer is responsible for creating the loadbalancers for API and API-INT endpoints for the cluster. These loadbalancers 
+are only created at install-time and their lifecycle is not owned by the cluster. Since the Kubernetes API is hosted in pods in the cluster, 
+using the Kubernetes service type `Loadbalancer` to manage the lifecycle of the loadbalancers is next-step for self-hosted Kubernetes.
+
+## Motivation
```

Comment:
This section fails to describe what actual enhancements the user will have.  IMO, we don't want the user managing the load balancers.  We want to set it up one way, right the first time, and the user doesn't touch it.  We're talking about the API endpoint, so performance should be a non-factor here.